client/lib/Widgets/LoginPage.dart

```diff
@@ -0,0 +1,387 @@
+  @override
+  LoginPageState createState() => new LoginPageState();
+}
+
+class LoginPageState extends State<LoginPage>{
+  final _username = TextEditingController();
+  final _password = TextEditingController();
+  final _forgottenUsername = TextEditingController();
+  bool _validateUsername = false;
+  bool _validatePassword = false;
+  static bool _validateForgottenUsername = false;
+  bool _isSelected = false;
+
+  void _radio(){
+    setState(() {
+      _isSelected = !_isSelected;
+    });
+  }
+
+  Widget radioButton(bool isSelected) => Container(
+    width: 16.0,
+    height: 16.0,
+    padding: EdgeInsets.all(2.0),
+    decoration: BoxDecoration(
+      shape: BoxShape.circle,
+      border: Border.all(width: 2.0, color: Colors.black),
+    ),
+    child: isSelected ? Container(
+      width: double.infinity,
+      height: double.infinity,
+      decoration: BoxDecoration(
+        shape: BoxShape.circle,
+        color: Colors.black,
+      ),
+    ) 
+    : Container(),
+  );
+
+  Widget horizontalLine() => Padding(
+    padding: EdgeInsets.symmetric(horizontal: 16.0),
+    child: Container(
+      width: ScreenUtil().setWidth(120),
+      height: 1.0,
+      color: Colors.black26.withOpacity(.2),
+    ),
+  );
+
+  void _showForgotPasswordDialog(BuildContext context) {
+    showDialog(
+      context: context,
+      builder: (context) {
+        return StatefulBuilder(builder: (context, setState) {
+          return AlertDialog(
+            title: Text('Enter your email address',
+              style: TextStyle(
+                fontFamily: "Poppins-Bold",
+                fontWeight: FontWeight.bold,
+              ),
+            ),
+            content: TextField(
+              controller: _forgottenUsername,
+              decoration: InputDecoration(
+                hintText: "Insert Email Address..",
+                hintStyle: TextStyle(
+                  color: Colors.grey, fontSize: 15.0,
+                ),
+                errorText: _validateForgottenUsername ? 'Email Address Can\'t Be Empty' : null,
+              ),
+            ),
+            actions: <Widget>[
+              FlatButton(
+                  child: Text(
+                    'Cancel',
+                    style: TextStyle(fontSize: 18),
+                  ),
+                  onPressed: () {
+                    setState(() {
+                      _forgottenUsername.clear();
+                    });
+                    Navigator.of(context).pop();
+                  }),
+              FlatButton(
+                child: Text(
+                  'Submit',
+                  style: TextStyle(fontSize: 18),
+                ),
+                onPressed: () {
+                  setState(() {
+                    _forgottenUsername.text.isEmpty ? _validateForgottenUsername = true : _validateForgottenUsername = false;
+                  });
+                  if(_forgottenUsername.text.isNotEmpty){
+                    Navigator.of(context).pop();
```

Comment:
I've used Flushbar to provide the message for both forgot password and for signing up.. Since I didn't have a Scaffold to base it off of. Updated pubspec.yaml for the Flushbar dependency